book/03_Geospatial_data_files/01_Geospatial_data_formats.md

```diff
@@ -0,0 +1,53 @@
+# Geospatial data formats
+
+## Vector data formats
+
+In this tutorial we will be using the following vector data formats:
+
+### 1. SHP
+
+Shapefile is the most widely known format for distributing geospatial data. It is a standard first developed by ESRI almost 30 years ago, which is considered ancient in software development.^3^ 
+
+The Shapefile consists of several files: in addition to one file with the actual geometry data, another file for defining the coordinate reference system is needed, as well as a file for defining the attributes and a file to index the geometries. This makes operating Shapefiles slightly clunky and confusing. However, Shapefile has been around for so long that any GIS software supports handling it.^3^
```

Comment:
It would be helpful to provide a little more detail here. That is, provide context that a user may need, e.g., are the files plain-text files or binary? Which files are mandatory and which are optional? (I believe most of this is described in https://en.wikipedia.org/wiki/Shapefile and in https://gisgeography.com/arcgis-shapefile-files-types-extensions/).